9.1/Dockerfile

```diff
@@ -49,7 +49,6 @@ RUN mkdir -p /var/run/postgresql && chown -R postgres /var/run/postgresql
 
 ENV PATH /usr/lib/postgresql/$PG_MAJOR/bin:$PATH
 ENV PGDATA /var/lib/postgresql/data
-VOLUME /var/lib/postgresql/data
```

Comment:
The entry point does this: https://github.com/plangrid/postgres/blob/master/docker-entrypoint.sh#L9